### PR TITLE
Show sign-in ads up after 3 edits and redesign the banner

### PR DESCRIPTION
**Pull request by danilowoz**
As part of the product growth strategy, from now on, the ads to sign-in will appears after the user takes 3 edits in the code.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5855/commits/af466b5bb68de117b446e0216630ce95319f336d is at https://pr5855.build.csb.dev/s/new.

**Comment by DannyRuchtie** [comment]
Should we make the entire text clickable?

**Comment by danilowoz** [comment]
Sure! Will do it

**Comment by tromika** [comment]
@danilowoz @DannyRuchtie After kicking the tires seems after 3 edits all the later edits trigger the banner that is actually overlaps the manu bar and the user has to click the x to close it. Should we use dismiss and try to be less intrusive?

File: packages/app/src/app/pages/Sandbox/Editor/Header/SignInAd.tsx [hunk 1]
```diff
@@ -1,115 +1,123 @@
...
+      /**
+       * A
+       */
+      if (experiment === ExperimentValues.A && bannerTriggerOnce === false) {
+        timer = window.setTimeout(() => {
+          setShow(true);
+          setBannerTriggerOnce(true);
+          // 3 minutes
+        }, 180000);
+      } else if (
```
Comment (danilowoz) [inline packages/app/src/app/pages/Sandbox/Editor/Header/SignInAd.tsx:33]: Create a new PR to test an A/A experiment before release

**Pull request closed without merging**